src/biogeochem/CNPhenologyMod.F90

```diff
@@ -125,11 +155,12 @@ subroutine CNPhenologyReadNML( NLFilename )
     integer :: ierr                 ! error code
     integer :: unitn                ! unit for namelist file
 
+    character(len=25) :: min_crtical_dayl_method   ! Method to determine critical day length for onset
```

Comment:
Should these be "critical" instead of "crtical" for consistency?

Reply:
OK, got that one.